### DISCUSSION TITLE
Server monitor

### DISCUSSION THREAD

**Discussion by mobeicanyue** [dani-garcia/vaultwarden#4240] [Q&A] [answered]
I disable the web front of vaultwarden, and visit it, it return 404, it is normal.
But now I want to monitor my vaultwarden service via status page, the status page visit my https format link of vault, still return 404(At frist I thouht it just return 404 writing in html.)
How can I do that? Is there any api I could use?

**Comment by BlackDex** [comment, marked as answer]
Use the `/alive` endpoint. That is also used by the Docker containers to check if the container is still working.
